test/benchmarks/benchmark.py

```diff
@@ -343,35 +343,56 @@ def main(args):
         "'python3 1gb_pytorch.py'",
     ]
 
-    # run in a loop
-    num_samples = 5
-    for x in range(len(jobs)):
-        print("Starting benchmarks for job {} with command {}".format(jobs[x], cmds[x]))
-        job = jobs[x]
-        for y in range(num_samples):
-            jobID = job + "-" + str(y)
-            process_stats = run_exec(cmds[x], jobID)
-            # wait a few seconds for memory to allocate
-            time.sleep(5)
-
-            # we don't mutate jobID for checkpoint/restore here so we can pass the unadulterated one to our csv
-            run_checkpoint(daemon_pid, jobID, output_dir, process_stats)
-            time.sleep(3)
-
-            run_restore(daemon_pid, jobID, output_dir)
-            time.sleep(3)
-
-            terminate_process(process_stats["pid"])
-
-    if local:
-        return
-    # unique uuid for blob id
-    blob_id = "benchmark-data-" + str(time.time())
-    push_otel_to_bucket("/cedana/data.json", blob_id)
-    attach_bucket_id("benchmark_output.csv", blob_id)
-    push_to_bigquery()
-    # delete benchmarking folder
-    cleanup()
+    if "--correctness" in args:
```

Comment:
I think we should eventually separate this from the benchmark.py script

Reply:
Lol kinda vague comment. Leaving a comment in here that we should separate it out is enough I think.

Reply:
yes i was thinking that too -- one top-level script that calls either benchmarking or correctness, so all the CI infra is shared